lib/srv/db/mongodb/protocol/opmsg.go

```diff
@@ -311,40 +310,3 @@ func (m *MessageOpMsg) ToWire(responseTo int32) (dst []byte) {
 	}
 	return bsoncore.UpdateLength(dst, idx, int32(len(dst[idx:])))
 }
-
-// ReadMsgSectionDocumentSequence reads multiple documents from the source.
-//
-// This function works in the same way as wiremessage.ReadMsgSectionDocumentSequence except that it
-// validate the rem index before reading the document.
-func ReadMsgSectionDocumentSequence(src []byte) (identifier string, docs []bsoncore.Document, rem []byte, ok bool) {
```

Comment:
This function is a wrapper to `wire.ReadMsgSectionDocumentSequence` with additional `if length-4 < 0` check. Since the same check is done in [validateDocumentSize](https://github.com/gravitational/teleport/pull/18690/files#diff-167af86fa58d3397c35c9678fad69267637aae8241db80e73845aed79b5e6a0eR291)  it is not longer needed.